command-line-local-shortener/templates/index.tmpl

```diff
@@ -0,0 +1,21 @@
+<!doctype html>
+<html>
+
+<head>
+    <title>Golang URL shortener</title>
+</head>
+
+<body>
+    <p>example paragraph<strong>example</strong> example
+        <strong>example</strong>example</p>
+
+    <form action="http://localhost:8090" method="POST">
```

Comment:
Should probably be something like `action="/create"`. The browser automatically assumes it's using the same domain name (ie. `http://localhost:8090`).